README.md

```diff
@@ -116,6 +116,13 @@ questions or comments.
 
     * machineset: Ensures that a minimum of two worker replicas are met.
 
+    * machinehealthcheck: Ensures the MachineHealthCheck resource is running as configured so that at most one worker node at a time is automatcially
```

Comment:
```suggestion
    * machinehealthcheck: Ensures the MachineHealthCheck resource is running as configured so that at most one worker node at a time is automatically
```